source/aks/patching-artifactory.html.md.erb

```diff
@@ -1,152 +1,111 @@
 ---
 title: Artifactory Patching
 weight: 30
-last_reviewed_on: 2024-03-13
+last_reviewed_on: 2024-06-06
 review_in: 12 months
 ---
 
 # <%= current_page.data.title %>
 
-Artifactory is utilized in the context of managing and updating software dependencies and configurations within a Kubernetes environment. Here's how it is used based on the provided instructions:
+Artifactory is utilized in the context of managing and updating software dependencies and configurations within a Kubernetes environment.
+Here's how it is used based on the provided instructions:
 
-The steps below will show you how to check the current version of Artifactory being used (7.71.5) and then to update it to a newer version (7.77.3). This process involves accessing Artifactory's website to determine the latest version available.
+The steps below will show you how to check the current **self-hosted version** of Artifactory being used and then to update it to a newer **self-hosted version**. This process involves accessing <[Artifactory's website](https://jfrog.com/help/r/jfrog-release-information/artifactory-release-notes)> to determine the latest **self-hosted version** version available.
 
-Before applying the update, a testing process is initiated. This involves downloading a repository containing configuration files for Artifactory (artifactory-sbox.yaml). This file is created and modified to reflect the desired changes, such as updating the version number.
+Before applying the update on PTL AKS Cluster, a testing process is initiated and is conducted on PTLSBOX AKS Cluster. This involves downloading the repository on <[GitHub](https://github.com/hmcts/cnp-flux-config)> containing configuration files for Artifactory and creating (artifactory-sbox.yaml). This file is then modified to reflect the desired changes, such as updating the version number.
```

Comment:
```suggestion
Before applying the update on PTL AKS Cluster, a testing process is initiated and is conducted on PTLSBOX AKS Cluster. This involves downloading the repository on [GitHub](https://github.com/hmcts/cnp-flux-config) containing configuration files for Artifactory and creating (artifactory-sbox.yaml). This file is then modified to reflect the desired changes, such as updating the version number.
```

I don't think you need these

Reply:
Good spot, my md checker had these added in and throwing errors all over the place when removed. seems doesn't need `<>` if `[]()` are together and not separated by a space.